packages/ui/src/kitchenSinkManager.tsx

```diff
@@ -18,6 +18,23 @@ export interface KitchenSinkManagerOptions {
     upperBarManager: UpperBarManager;
 }
 
+export enum ItemActiveStates {
+    Active = "Active",
+    InActive = "InActive"
+}
+
+export enum EventTypes {
+    ItemActiveStateChangeEvent = "ItemActiveStateChangeEvent"
+}
+
+export interface ItemActiveStateChangeEvent {
+    type: EventTypes.ItemActiveStateChangeEvent;
+    state: ItemActiveStates;
+    item: KitchenSinkItem;
+}
+
+type Events = ItemActiveStateChangeEvent;
```

Comment:
consider moving all the event related artifacts to `kitchenSinkEvents.ts`,  might be not relevant, wondering what you are thinking about it.

Reply:
might be nice... will do it if additional events will be added...